### ISSUE TITLE
A problem about flow_bus_matrix method.

### ISSUE THREAD

**Issue by wzz-wzz-wzz** [rte-france/Grid2Op#220]
## Environment
 - Grid2op version: `1.5.2`
 - System: ` ubuntu18.04`

## Bug description

In the l2rpn_case14_sandbox environment, if  I perform an action related to disconnection, the flow bus matrix can not be obtained, and because of this,  I can not get the networkx graph either.

### Code snippet

```python
import grid2op
env_name = 'l2rpn_case14_sandbox'
env = grid2op.make(env_name)
obs = env.reset()
action = env.action_space()
action.change_line_status = [9]
new_obs, *_ = env.step(action)
flow_matrix = new_obs.flow_bus_matrix()

```

## Current output

```
ValueError: row, column, and data array must all be the same length
```

## Expected output

```
I want to get a flow bus matrix or networkx graph when I perform an action related to disconnection. Thanks a lot!
```

**Comment by BDonnot** [comment]
Hello,

Thanks for reporting this issue. I just fixed in the branch development. It will be included in the next release (coming soon, this week or the next)

**Comment by BDonnot** [comment]
The bugfix is available in the "development" version of grid2op that you can install with:
`pip install git+git://github.com/rte-france/grid2op.git@dev_1.6.0`

**Comment by wzz-wzz-wzz** [comment]
Oh, thank you very much! I just tried this new version and  it worked! @BDonnot

**Comment by BDonnot** [comment]
I am reoppening this issue because the fix is not pushed on pypi (yet) and other people can encounter this issue with version 1.5.2 (last stable release).

I will close it after the fix is stable :-)

**Comment by wzz-wzz-wzz** [comment]
OK, I understand. Thanks again! @BDonnot

**Comment by BDonnot** [comment]
Fixed and merged in version 1.6.0, now available on pypi
